docs/tutorials/8_train_custom_env.ipynb

```diff
@@ -0,0 +1,494 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[download this notebook here](https://github.com/HumanCompatibleAI/imitation/blob/master/docs/tutorials/8_train_custom_env.ipynb)\n",
+    "# Train Behavior Cloning in a Custom Environment\n",
+    "\n",
+    "You can use `imitation` to train a policy in a custom environment.\n",
+    "Here, we re-implement a [fixed-horizon](https://imitation.readthedocs.io/en/latest/getting-started/variable-horizon.html) variant of the CartPole environment (also available in [seals](https://github.com/HumanCompatibleAI/seals)), and go through the steps of training a policy using behavior cloning in that environment.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 1: Define the environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we need to define our custom environment. We'll use the same dynamics as the original CartPole environment, but remove the termination condition, so that the environment has a fixed horizon.\n",
+    "\n",
+    "If you have your own environment that you'd like to use, you can replace the code below with your own environment. Make sure it complies with the standard Gym API, and that the observation and action spaces are specified correctly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import gym\n",
+    "\n",
+    "from gym.spaces import Discrete, Box\n",
+    "from gym.utils import seeding\n",
+    "\n",
+    "\n",
+    "class FixedHorizonCartPoleEnv(gym.Env):\n",
```

Comment:
This is quite a complex environment just for a tutorial, a user could get lost in some of these details. Consider something even more toy?